Configuration/AlCa/python/autoCond.py

```diff
@@ -83,7 +83,7 @@
     # GlobalTag for MC production with realistic conditions for Phase1 2024
     'phase1_2024_realistic'    : '113X_mcRun3_2024_realistic_v4', # GT containing realistic conditions for Phase1 2024
     # GlobalTag for MC production with realistic conditions for Phase2
-    'phase2_realistic'         : '113X_mcRun4_realistic_v3'
+    'phase2_realistic'         : '113X_mcRun4_realistic_Candidate_2021_02_11_17_34_23'
```

Comment:
Hi @Sam-Harper is this GT change wanted or just a mistake?I don't think a candidate GT can be put in the release, but maybe @christopheralanwest can confirm?

Reply:
my understanding is that @christopheralanwest will edit the PR with the correct GT now that its all approved.

Reply:
It's fine to use a GT candidate in PR tests, just as long as it is converted to a versioned GT prior to merging. I created the versioned GT, which is equivalent to your candidate:

https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/113X_mcRun4_realistic_Candidate_2021_02_11_17_34_23/113X_mcRun4_realistic_v4

Please update autoCond with the new GT.

Reply:
thanks Chris, done!